shoop/core/models/_orders.py

```diff
@@ -260,6 +261,7 @@ class Order(MoneyPropped, models.Model):
     require_verification = models.BooleanField(default=False, verbose_name=_('requires verification'))
     all_verified = models.BooleanField(default=False, verbose_name=_('all lines verified'))
     marketing_permission = models.BooleanField(default=True, verbose_name=_('marketing permission'))
+    _codes = JSONField(blank=True, null=True, verbose_name=_('codes'))
```

Comment:
Yeah, and `Order.codes` can't be set to a dict for example, since a freely accessible JSONField would allow all kinds of misuse.